Makefile

```diff
@@ -250,6 +251,10 @@ run: clear
 		"cd ./src; \
 		uvicorn $(PACKAGE_NAME).main:app $(RELOAD) --host 0.0.0.0 --port $(SERVER_PORT_CONTAINER);"
 
+ingest:
+	$(info Running ingestion script to populate database...)
+	export JUPYTER_PORT=$(JUPYTER_PORT); export CONTAINER_USER=$(CONTAINER_USER); export UID=$(UID); export GID=$(GID); docker-compose -p $(APP_NAME) --env-file ./.env run --entrypoint="bash -c \"pip install httpx pandas; cd /home/$(CONTAINER_USER); jupyter nbconvert --to script PopulateDatabase.ipynb; cat PopulateDatabase.py | grep -v get_ipython > run.py; python run.py; rm PopulateDatabase.py run.py\"" jupyter
```

Comment:
Why's this going via the entrypoint rather than being run as a command?

Reply:
Because the entrypoint currently starts a jupyter notebook server with all sorts of quality of life improvements (e.g. dark mode) which takes time. We don't need that at all for a command line script, I just wanted to void the original entrypoint hence overwriting it.

Reply:
Looks like the _command_ does, but the entry point isn't specified so it'll otherwise be the image's defaul, which is https://github.com/jupyter/docker-stacks/blob/main/docker-stacks-foundation/start.sh

Reply:
You're right. Changing that.